lxd/metrics/api_rates.go

```diff
@@ -79,6 +81,11 @@ func GetCompletedRequests(entityType entity.Type, result RequestResult) int64 {
 func TrackStartedRequest(r *http.Request, endpointType entity.Type) {
 	// Set the callback function to track the request as completed.
 	// Use sync.Once to ensure it can be called at most once.
+	if !shared.ValueInSlice(endpointType, entity.APIMetricsEntityTypes()) {
```

Comment:
Is it possible to enter this block? I thought that all metrics entity types were checked at start up when the server is initialised?

Reply:
It is possible if you use `TrackStartedRequest` on an endpoint that is not supposed to be counted. Or at least that was the reasoning behind this, if you think it is not needed it is fine for me :)

Reply:
Yeh lets remove this for now.

Reply:
Done!